tests/integration/test_system_containers_images.sh

```diff
@@ -88,11 +93,37 @@ ${ATOMIC} images prune
 ${ATOMIC} images list -f type=ostree --all > ${WORK_DIR}/images.all.out
 assert_matches "<none>" ${WORK_DIR}/images.all.out
 
-# Test that the image can be deleted by ID
+#Test that when image is being used, it can not be deleted without force option
+${ATOMIC} install --system -n ${CONTAINER_NAME} docker.io/busybox
+${ATOMIC} --assumeyes images delete --storage ostree docker.io/busybox > ${WORK_DIR}/image.delete.out
+assert_matches "YES" ${WORK_DIR}/image.delete.out
+assert_matches "unable to remove repository reference" ${WORK_DIR}/image.delete.out
+assert_matches ${CONTAINER_NAME} ${WORK_DIR}/image.delete.out
+assert_matches "docker.io/busybox" ${WORK_DIR}/image.delete.out
+${ATOMIC} --assumeyes containers delete ${CONTAINER_NAME}
+
+# Test that when the named layer is still present, we are unable to delete the unnamedlayer
+UNNAMED_LAYER_ID=$(${ATOMIC} images list -a -f type=ostree| grep '<none>' | awk '{print $3}')
```

Comment:
This is probably fine, but I still caution against doing this because you're assuming the busybox image will always have 1 unnamed layer.